src/library/searchquery.cpp

```diff
@@ -166,6 +172,46 @@ QString TextFilterNode::toSql() const {
     return concatSqlClauses(searchClauses, "OR");
 }
 
+CrateFilterNode::CrateFilterNode(const QSqlDatabase& database,
+                                 const QString& argument)
+    : m_database(database),
+      m_argument(argument) {
+    getTrackIds();
+}
+
+void CrateFilterNode::getTrackIds() {
+    FieldEscaper escaper(m_database);
+    QString escapedArgument = escaper.escapeString(kSqlLikeMatchAll + m_argument + kSqlLikeMatchAll);
+
+    QSqlQuery query(m_database);
+    QString queryString = CrateStorage::formatQueryForTrackIdsByCrateName(escapedArgument);
+    query.setForwardOnly(true);
+    query.prepare(queryString);
+
+    if (query.exec()) {
+        if (sDebug) {
```

Comment:
This conditional would only be necessary if formatting of the debug log message is expensive -> Remove if()